### PR TITLE
zib-HealthProfessional pattern profiles

### PR DESCRIPTION
**Pull request by pieter-edelman-nictiz**
Even een nieuwe PR om de nieuwe pattern-profielen te beoordelen.

### CODE REVIEW

File: resources/nl-core/pattern-NlCoreHealthProfessionalReference.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="pattern-NlCoreHealthProfessionalReference" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/pattern-NlCoreHealthProfessionalReference" />
+  <name value="PatternNlCoreHealthProfessionalReference" />
+  <title value="Pattern for zib HealthProfessional References" />
```
Comment (ArdonToonstra) [inline resources/nl-core/pattern-NlCoreHealthProfessionalReference.xml:6]: Should the name be Pattern for **nl-core** HealthProfessional References?

**Review by ArdonToonstra — COMMENTED** [review]
I updated this for MedicationContraIndication: seems to work as we want to, right?
👯 
image

File: resources/nl-core/pattern-NlCoreHealthProfessionalReference.xml (hunk 1 above)
Comment (pieter-edelman-nictiz) [reply resources/nl-core/pattern-NlCoreHealthProfessionalReference.xml:6]: I'd rather stick with zib, as this is about references to the zib, not the profile that implements it.

File: resources/zib/ext-CodeSpecification.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    <type value="element" />
+    <expression value="code" />
+  </context>
+  <context>
+    <type value="element" />
+    <expression value="Coding" />
+  </context>
+  <context>
+    <type value="element" />
+    <expression value="CodeableConcept" />
```
[the remark this answers is not in the document]
Comment (pieter-edelman-nictiz) [reply resources/zib/ext-CodeSpecification.xml:32]: Het is ook niet best dat hetzelfde ding in meerder branches leeft.

File: resources/nl-core/pattern-NlCoreHealthProfessionalReference.xml (hunk 1 above)
Comment (ArdonToonstra) [reply resources/nl-core/pattern-NlCoreHealthProfessionalReference.xml:6]: Ok!

**Pull request closed without merging**